Makefile

```diff
@@ -24,11 +24,16 @@ compile-tests: $(TEST_SRC)
 	mkdir -p build && cd build && $(CC) $(CFLAGS) $(TEST_LIBOBJ) $? -o tax_test
 	./build/tax_test | sed -n -e 's/^.*test\/\([^\S]*\):\([^\D]*\).*$$/test\/\1:\2/p' -e 's/-----------------------/-----------------------/p' -e 's/OK/OK/p' -e 's/FAIL/FAIL/p' -e 's/\([d]* Tests\)/\1/p' > test.log
 
+tests: all compile-tests
+
+test: tests
+
+.PHONY: docs clean
+
 docs:
+	rm -rf docs/*
```

Comment:
Is there any change in the user/developer instructions due to make file changes?